MCprep_addon/world_tools.py

```diff
@@ -993,7 +999,7 @@ def execute(self, context):
 		self.track_param = engine
 		return {'FINISHED'}
 
-	def create_sunlamp(self, context):
+	def create_sunlamp(self, context: Context) -> Object:
```

Comment:
To prevent confusion
```suggestion
	def create_sunlamp(self, context: Context) -> bpy.types.Object:
```